src/script/bitcoinconsensus.cpp

```diff
@@ -69,7 +69,7 @@ struct ECCryptoClosure
 ECCryptoClosure instance_of_eccryptoclosure;
 }
 
-int bitcoinconsensus_verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, uint64_t amount,
+int bitcoinconsensus__verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, CAmount amount,
```

Comment:
Meant to mention that; it can be anything, I just don't think it should be exposed via the API.